controllers/actions.go

```diff
@@ -171,6 +171,7 @@ func (r *ClusterGroupUpgradeReconciler) deleteResources(
 		return fmt.Errorf("failed to delete precaching objects for CGU %s: %v", clusterGroupUpgrade.Name, err)
 	}
 
+	// comment: should we cleanup in failed clusters as well?
```

Comment:
@jc-rh @vitus133, do we only delete resources where precaching and backup were successful, leaving the failed on for debugging or de we cleanup the objects in all the clusters?

Reply:
I belive we had determined before that leaving such resources do not actually aid in debugging